src/components/discardChanges.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <div class="container">
+        <header class="header">
+            <div class="header-controls">
+                <button 
+                    @click="discardChanges" 
+                    class="btn-secondary btn" 
+                    :class="{'disabled': !wasSomethingChanged}">
```

Comment:
We also have to disable the button itself when the `wasSomethingChanged` is false